Makefile

```diff
@@ -55,18 +55,26 @@ helm-test:
 helm-lint:
 	cd config/helm && ./testing/lint.sh
 
+create-kind-cluster:
+	kind create cluster --config ./src/testing/kuttl/oneagent/kind.yaml
+
+delete-kind-cluster:
+	kind delete cluster --name kind
 
 kuttl-install:
 	hack/e2e/install-kuttl.sh
 
-kuttl-oneagent: deploy
+kuttl-oneagent: kuttl-check-mandatory-fields deploy
 	kubectl -n dynatrace wait pod --for=condition=ready -l app.kubernetes.io/component=webhook
 	kubectl kuttl test --config src/testing/kuttl/oneagent/oneagent-test.yaml
-# CLEAN-UP
+	# CLEAN-UP
 	kubectl delete dynakube --all -n dynatrace
-	kubectl -n dynatrace wait pod --for=delete -l app.kubernetes.io/component=oneagent --timeout=500s
+	-kubectl -n dynatrace wait pod --for=delete -l app.kubernetes.io/component=oneagent --timeout=500s
 	kubectl delete -f config/deploy/kubernetes/kubernetes-all.yaml
 
+kuttl-check-mandatory-fields:
+	hack/do_env_variables_exist.sh "APIURL APITOKEN PAASTOKEN"
```

Comment:
Plz make is a first assert in kuttl and remove from Makefile

Reply:
why would we need to assert that in every test case? The tests failing would be indication enough, wouldn't they?

Reply:
The test failing would be enough, there would not be an advantage for pipeline tests

But it is a great advantage for manual testing, as everytime Michal or I started the tests we forgot to the set the variables  => therefore had to wait to get an error and wonder why the test failed.

So in my opinion, we should check that, even if the tests would be sufficient.